### PR TITLE
internal: Fix typos.

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

**Comment by jayvdb** [comment]
@milesj the previous failure in CI now passes locally
`cargo run -- --color --log trace ci --base master`

The docker workflow failure looks like probably something unrelated to this PR.
https://github.com/moonrepo/moon/actions/runs/11042801333/job/30676248537?pr=1665
I'm happy to investigate that further if you'd like.

**Comment by milesj** [comment]
@jayvdb I think it's because of a fork.

To fix the version check, you can run `just bump patch`.

File: crates/action-graph/src/action_graph.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ impl ActionGraph {
         ActionGraph { graph }
     }
 
-    pub fn creater_iter(&self, indices: Vec<NodeIndex>) -> ActionGraphIter {
+    pub fn create_iter(&self, indices: Vec<NodeIndex>) -> ActionGraphIter {
```
Comment (jayvdb) [inline crates/action-graph/src/action_graph.rs:26]: this should probably require a major release?

I can revert it if that is undesirable?

**Comment by jayvdb** [comment]
I tried that, but there wasnt any change

```
moon> just bump patch
bash ./scripts/version/bumpBinaryVersions.sh patch
packages/cli -> patch
packages/core-linux-arm64-gnu -> patch
packages/core-linux-arm64-musl -> patch
packages/core-linux-x64-gnu -> patch
packages/core-linux-x64-musl -> patch
packages/core-macos-arm64 -> patch
packages/core-macos-x64 -> patch
packages/core-windows-x64-msvc -> patch

moon> git status
On branch typos
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        .typos.toml
        .yarn/versions/

nothing added to commit but untracked files present (use "git add" to track)
```

The change to `@moonrepo/types@workspace:packages/types` probably doesnt need to be released.

I also tried

```
moon> yarn version check --interactive
Press <up>/<down> to select workspaces.                      Press <enter> to save.
Press <left>/<right> to select release strategies.           Press <ctrl+c> to abort.

The following files have been modified in your local checkout.

  /home/jayvdb/rust/moon/.typos.toml
  /home/jayvdb/rust/moon/.yarn/versions/186654ce.yml
  /home/jayvdb/rust/moon/crates/action-graph/src/action_graph.rs
  /home/jayvdb/rust/moon/crates/action-graph/src/action_graph_builder.rs
  /home/jayvdb/rust/moon/crates/action-graph/tests/action_graph_test.rs
  /home/jayvdb/rust/moon/crates/codegen/src/codegen.rs
  /home/jayvdb/rust/moon/crates/codegen/src/codegen_error.rs
  /home/jayvdb/rust/moon/crates/config/src/project/dep_config.rs
  /home/jayvdb/rust/moon/crates/config/src/project_config.rs
  /home/jayvdb/rust/moon/crates/project-builder/src/project_builder.rs
  /home/jayvdb/rust/moon/crates/task-runner/src/command_executor.rs
  /home/jayvdb/rust/moon/legacy/node/lang/src/pnpm/dependency_path.rs
  /home/jayvdb/rust/moon/packages/types/src/project-config.ts

Because of those files having been modified, the following workspaces may need to be released again (note that private workspaces are also shown here, because even though they
won't be published, releasing them will allow us to flag their dependents for potential re-release):

1 total, 1 release, 0 remaining

 > @moonrepo/types@workspace:packages/types - 1.20.0 → 1.20.1
     ◯ undecided  ◯ decline  ◉ patch  ◯ minor  ◯ major

The following workspaces depend on other workspaces that have been marked for release, and thus may need to be released as well:
(Press <tab> to move the focus between the workspace groups.)

   @moonrepo/nx-compat@workspace:packages/nx-compat - 0.0.2
     ◯ undecided  ◉ decline  ◯ patch  ◯ minor  ◯ major
   @moonrepo/report@workspace:packages/report - 1.3.1
     ◯ undecided  ◉ decline  ◯ patch  ◯ minor  ◯ major
   @moonrepo/runtime@workspace:packages/runtime - 0.0.16
     ◯ undecided  ◉ decline  ◯ patch  ◯ minor  ◯ major
   website@workspace:website - 0.1.0 → 0.1.1
     ◯ undecided  ◯ decline  ◉ patch  ◯ minor  ◯ major
```

But again nothing changed.

File: crates/action-graph/src/action_graph.rs (hunk 1 above)
Comment (milesj) [reply crates/action-graph/src/action_graph.rs:26]: We don't publish these crates so it's fine.

**Comment by milesj** [comment]
@jayvdb You need to commit the `.yarn/versions` file after running the bump.

**Comment by milesj** [comment]
Awesome thanks a bunch!

**Pull request merged**